sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -573,6 +573,7 @@ final class DataFrameWriter[T] private[sql](ds: Dataset[T]) {
    * indicates a timestamp format. Custom date formats follow the formats at
    * `java.text.SimpleDateFormat`. This applies to timestamp type.</li>
    * </ul>
+   * <li>`encoding`(default `utf-8`) save dataFrame 2 csv by giving encoding</li>
```

Comment:
Could we just resemble the documentation in `DataFrameReader` just for consistency?

```
 <li>`encoding` (default `UTF-8`): decodes the CSV files by the given encoding
   * type.</li>
```